samples/multiplatform/src/jsTest/mocha/src/test/kotlin/ch/tutteli/SmokeTest.kt

```diff
@@ -0,0 +1,68 @@
+package ch.tutteli
+
+import ch.tutteli.atrium.api.fluent.en_GB.isLessThan
```

Comment:
It would have been enough for me if you add one JS runner but good as well.
Feel free to remove one in case you get problems (once you setup the runners to actually execute the tests)